cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -558,14 +571,15 @@ EOS
 end
 
 datasource "ds_azure_databricks_managed_vm_metrics" do
+  batch true
   iterate $ds_virtualmachines_from_bill
   request do
-    run_script $js_azure_databricks_managed_vm_metrics, $param_azure_endpoint, $ds_param_stats_interval, $ds_lookback_time_formats, jq(iter_item, "[.virtualmachines[] .resourceId ]"), jq(iter_item, ".databricks_workspace.location"), jq(iter_item, ".databricks_workspace.managedResourceGroupId")
+    run_script $js_azure_databricks_managed_vm_metrics, $ds_param_stats_interval, $ds_lookback_time_formats, $param_azure_endpoint, jq(iter_item, "[.virtualmachines[] .resourceId ]"), jq(iter_item, ".databricks_workspace.location"), jq(iter_item, ".databricks_workspace.managedResourceGroupId")
```

Comment:
Variables listed in the wrong order - "run_script parameters should be in the following order: script, val(iter_item, string), datasources, parameters, variables, raw values:"

And remember to reflect the same on Line 582